packages/plugin-electron-deliver-minidumps/package.json

```diff
@@ -18,7 +18,8 @@
     "send-minidump.js"
   ],
   "dependencies": {
-    "form-data": "^4.0.0"
+    "form-data": "^4.0.0",
+    "@bugsnag/electron-network-status": "^7.10.0"
```

Comment:
We've had issues with putting our own packages in dependencies because it allows you to end up with multiple versions of the same package installed at the same time. Mostly this just causes TS errors so it's probably fine in this case, but usually we'd make this a peer dependency of `plugin-electron-deliver-minidumps` and `delivery-electron`, with the main `@bugsnag/electron` package pulling it in as a real dependency

Some background on PLAT-4872